### PR TITLE
Add new tutorial - Install OpenStack with conjure-up

### LINKED ISSUE #453: Tutorials Wanted - Install single-server Openstack with conjure-up
This tutorial will show the user how to deploy OpenStack on a single (well-specced!) machine, using the conjure-up spell.

There is already a video walkthrough of this - https://www.ubuntu.com/download/cloud/try-openstack

### PR DESCRIPTION
**Pull request by konrad11901**
Add new tutorial for installing single-server OpenStack with conjure-up.

Fixes #453.

### CODE REVIEW

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+author: Konrad Krawiec <konrad11901@gmail.com>
+
+---
+
+# Install single-server OpenStack with conjure-up
+
+## Overview
+Duration: 1:00
+
+[OpenStack](https://www.openstack.org) is a very popular software for creating private and public clouds.
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: *software* here doesn't quite make sense. Perhaps replace with *option* or some expansion of what kind of software it us, such as *software stack*.

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+OpenStack is quite an advanced platform and deploying it may not be an easy task. Luckily, with [conjure-up](https://conjure-up.io), it's very simple and straightforward.
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: I'd remove *quite* and add maybe *deploying ***manually*** may not be an easy task*

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+### What you'll learn
+
+- How to install OpenStack on your machine with conjure-up
+- How to install and configure LXD
+
+### What you'll need
+
+- A well-specced computer running Ubuntu 16.04 LTS with 16 gigabytes of RAM (SSD drive is strongly recommended)
+- Some basic command-line knowledge
+- An hour of your time
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: I really like 'An hour of your time' :+1:

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+We'll need to set up [LXD](https://linuxcontainers.org/lxd) to deploy OpenStack.
+
+negative
+: **LXD version**
+conjure-up requires that the minimum version of LXD be **2.18**, so the version of LXD distributed through official repositories (which can be installed with `sudo apt install lxd`) is **not** compatible with conjure-up.
+
+### Installing LXD
+
+The recommended installation method of LXD are [snaps](https://snapcraft.io). Don't worry - the snap support is included with Ubuntu 16.04 LTS and above, so we don't have to do anything.
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: I'd write something more like *The recommended LXD installation method is to use snaps...*

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+sudo snap install conjure-up --classic
+```
+
+Then, simply run:
+
+```bash
+conjure-up
+```
+
+We'll see the "Spell selection" screen.
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: End with a colon?

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+![](./images/conjure-up-2.png)
+
+On the next screen, leave the default values and click "Save":
+
+![](./images/conjure-up-3.png)
+
+positive
+: **Navigating inside conjure-up**
+To navigate between the main content and buttons located at the bottom of the screen, use **Tab**.
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: ^ Excellent advice!

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+It may take several minutes for the deployment to finish. **Please be patient**.
+
+Next, the post-deploy steps will be run:
+
+![](./images/conjure-up-8.png)
+
+Again, please be patient and wait for the post-deploy steps to finish.
+
+After everything is completed, we'll see the following screen:
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: *has* completed.

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+![](./images/conjure-up-9.png)
+
+Copy the Horizon URL to the clipboard or store it somewhere - we'll need it in a while.
+
+## Checking if OpenStack is working
+Duration: 1:00
+
+Now, let's check if everything is working.
+
+Open a web browser and navigate to the Horizon using the URL we copied before.
```
Comment (degville) [inline tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: add something like*You will see the following login screen:*

File: tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md (hunk 3 above)
Comment (konrad11901) [reply tutorials/containers/install-openstack-with-conjure-up/install-openstack-with-conjure-up.md]: To be honest, in my case it was like two hours or so 😆

**Comment by konrad11901** [comment]
Thanks for your review! I'll make the requested changes shortly.

**Pull request merged**